packages/admin/admin-react-select/src/ReactSelect.tsx

```diff
@@ -160,9 +157,9 @@ const components = {
 export interface SelectProps<OptionType extends OptionTypeBase> {
     theme: Theme;
     selectComponent: React.ComponentType<ReactSelectProps<OptionType>>;
-    clearIcon?: SvgIconComponent;
-    dropdownIcon?: SvgIconComponent;
-    dropdownIconOpen?: SvgIconComponent;
+    clearIcon?: React.ReactNode;
+    dropdownIcon?: React.ReactNode;
+    dropdownIconOpen?: React.ReactNode;
```

Comment:
I don't think this type is correct: This would require `<Clear />`, but the usage in the component suggests `Clear`. Please try this in Storybook

Reply:
There was no Typescript error but I changed it nevertheless to make it more specific:
https://github.com/vivid-planet/comet/pull/2111/commits/1c3834a98ae9e26493b2ebfad126472e6291fb06